safe/common/parameters/default_select_parameter_widget.py

```diff
@@ -151,3 +152,29 @@ def toggle_custom_value(self):
             self.custom_value.setDisabled(False)
         else:
             self.custom_value.setDisabled(True)
+
+    def toggle_input(self):
+        """Change behaviour of radio button based on input."""
+        # If input is a field, then disable all radio button.
+        if self.input.currentText() != self.input.itemData(0, Qt.UserRole):
```

Comment:
Yes, I'm sorry. My intention of doing this back then is I just want to update my remote repo with my local repo. But I also wondering about this. I think by creating an attribute, it will still checking the current combo box text or the itemData anyway. Isn't it?